terraform/aks-cluster/main.tf

```diff
@@ -31,7 +31,6 @@ locals {
   api_server_authorized_ip_ranges = [
     "0.0.0.0/0"
   ]
-
 }
 
 resource "azurerm_kubernetes_cluster" "adl_aks" {
```

Comment:
**[tfsec]** <[AVD-AZU-0041](https://aquasecurity.github.io/tfsec/v1.28.1/checks/azure/container/limit-authorized-ips/)> <sub>reported by [reviewdog](https://github.com/reviewdog/reviewdog) :dog:</sub><br>Cluster does not limit API access to specific IP addresses.